LiveWallpaperQuotes/app/src/main/java/com/stanleyidesis/quotograph/ui/debug/DebugDialog.java

```diff
@@ -0,0 +1,110 @@
+package com.stanleyidesis.quotograph.ui.debug;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.text.format.DateUtils;
+import android.util.Log;
+import android.view.View;
+
+import com.afollestad.materialdialogs.MaterialDialog;
+import com.stanleyidesis.quotograph.LWQApplication;
+import com.stanleyidesis.quotograph.LWQPreferences;
+import com.stanleyidesis.quotograph.RemoteConfigConst;
+import com.stanleyidesis.quotograph.ui.activity.LWQSurveyActivity;
+import com.stanleyidesis.quotograph.ui.dialog.SurveyDialog;
+
+/**
+ * Copyright (c) 2016 Stanley Idesis
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ *
+ * DebugDialog.java
+ * @author Stanley Idesis
+ *
+ * From Quotograph
+ * https://github.com/stanidesis/quotograph
+ *
+ * Please report any issues
+ * https://github.com/stanidesis/quotograph/issues
+ *
+ * Date: 09/11/2016
+ */
+public class DebugDialog {
```

Comment:
Make this nicer in the future, less hard-coded and perhaps easier to use than a long list of options?